app/src/main/java/com/scrumptious/algorithmvisualizer/SelectionSortSequenceGen.kt

```diff
@@ -0,0 +1,122 @@
+package com.scrumptious.algorithmvisualizer
+
+
+class SelectionSortSequenceGen {
+fun selectionSortSequenceGenerator(arr: Array<Int>): Array<Pair<Array<Int>, Array<Int>>> {
```

Comment:
This needs the result attribute in order to be consistent with the Class Diagram